mne/viz/_3d.py

```diff
@@ -595,9 +595,10 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
     seeg : bool
         If True (default), show sEEG electrodes.
     fnirs : str | list | bool | None
-        Can be "channels" or "pairs" to show the fNIRS channel locations or
-        line between source-detector pairs, or a combination like
-        ``('pairs', 'channels')``. True translates to ``('pairs',)``.
+        Can be "channels", "pairs", "detectors", and/or "sources" to show the
+        fNIRS channel locations, optode locations, or line between
+        source-detector pairs, or a combination like ``('pairs', 'channels')``.
+        True translates to ``('pairs')``.
```

Comment:
```suggestion
        True translates to ``('pairs',)``.
```
There is a difference, what you wrote is actually a `str` and the suggested change is a `tuple` with one `str` element

Reply:
fixed